### PR TITLE
Update readme with basic build and run steps

### PR DESCRIPTION
**Pull request by rikgibson**
Some very minor changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 [![wercker status](https://app.wercker.com/status/eeb2240b621c0181c460d73a18971de2/s "wercker status")](https://app.wercker.com/project/bykey/eeb2240b621c0181c460d73a18971de2)
 
 A sample application in Go for Wercker.
 
-This application uses the `golang` container obtained from the [Docker Hub](https://hub.docker.com/r/_/golang/)
+This application uses the `golang` container obtained from the [Docker Hub](https://hub.docker.com/_/golang/)
 
 ## Setup & Build
-Clone this repo and cd into the directory:
+Clone or fork this repo and cd into the directory:
```
Comment (jwolski2) [inline README.md]: Leave this line as-is. "Forking" seems irrelevant. Even if you choose to _fork_, cloning is still necessary.

Comment (rikgibson) [reply README.md]: So the devcenter docs use the phrase "fork and clone" - "Before we can start developing, we have to fork and clone the ..."

Suggest using consistent language in both places.

Comment (jwolski2) [reply README.md]: Sgtm. Go with "fork and clone" (but not "clone or fork")
